jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -569,6 +569,15 @@ def test_default_spectral_extraction(cubeviz_helper, spectrum1d_cube_fluxunit_jy
         extracted_spectra[0].flux, extracted_spectra[1].flux
     )
 
+    # Also test unit conversion
+    uc = cubeviz_helper.plugins["Unit Conversion"]._obj
+    uc.flux_unit_selected = "MJy"
+    spec_extr_plugin = cubeviz_helper.plugins['Spectral Extraction']._obj
+    collapsed = spec_extr_plugin.extract()
+    assert collapsed.flux.unit == u.MJy
+    assert collapsed.uncertainty.unit == u.MJy
+    assert_allclose(collapsed.flux.value, extracted_spectra[0].flux.value / 1e6)
+
```

Comment:
```suggestion
assert uc.flux_unit.selected == 'MJy'
```

Reply:
The indent in suggestion is wrong but I added it in a follow-up commit with correct indent. Thanks!